src/app/components/accordion/accordion.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="accordion__wrapper" [ngClass]="{'accordion__content-reverse': isReverse}">
+
+<button (click)="toggleVisibility()" class="accordion__button">
+  <span *ngIf="buttonName">{{ buttonName }}</span>
+  <img
+    *ngIf="isIcon"
+    class="accordion__arrow-icon"
+    [ngClass]="{ rotate: isOpen }"
+    [src]="iconsRoute + 'chevron-down.svg'"
+    alt="lefelé mutató nyíl"
```

Comment:
kellene ide nyelvesites nem?

Reply:
es nem is lefele mutato mert fogatjuk adj neki jobb nevet

Reply:
ehhez mit szólsz?
[alt]="'arrow' + isOpen ? 'up' : 'down'"